### PR TITLE
Club: post list layout refinement

### LINKED ISSUE #6221: club: Templates: Search, Category, Tag
Implement Search, Category, Tag pages following the mockups.
image

### PR DESCRIPTION
**Pull request by matiasbenedetto**
#### Changes proposed in this Pull Request:
This PR is a followup from https://github.com/Automattic/themes/pull/6236. In this one we are refining the implementation of the post list responsive layout to match the theme mockups.

:warning: Consider in the review just the layout the position of the elements, not the typography sizes that is part of a different PR.

Screen capture:

https://user-images.githubusercontent.com/1310626/179822318-9735efdc-d8c0-4c68-9cb1-8c17c2492df7.mp4

Fixes: #6221

### CODE REVIEW

**Comment by madhusudhand** [comment]
I don't see it working in responsive mode.

https://user-images.githubusercontent.com/1935113/180386044-92ecc249-94e1-43d5-9a99-4e0a391e04ef.mp4

**Update:** Ah, class isn't there in the frontend.

image

**Comment by vcanales** [comment]
In my case, it works on Firefox

|Responsive|Not Responsive|
|---|---|
|image|image|

Safari looks a bit wonky while not responsive

|Responsive|Not Responsive|
|---|---|
|image|image|

**Comment by matiasbenedetto** [comment]
> I don't see it working in responsive mode.

@madhusudhand I think you need to remove your local changes to the template to be able to use the template from the HTML file. Please try again after doing that, please.

**Comment by matiasbenedetto** [comment]
> Safari looks a bit wonky while not responsive

I'm not able to test in Safari right now (I'm running Linux), but using a Webkit (the same engine as Safari) browser I get this:

https://user-images.githubusercontent.com/1310626/181072954-2da8bd5b-2ab6-4012-9a79-d6f539adb92e.mp4

I will try to get a macOs emulator or pair with someone running macOs.

**Comment by jffng** [comment]
I have the same result as @vcanales when testing on Safari: 

Screen Shot 2022-08-01 at 11 07 45 AM

Unsetting `min-width` in the column's CSS fixes the issue for me:

Screen Shot 2022-08-01 at 11 09 06 AM

**Comment by matiasbenedetto** [comment]
I removed min-width as @jffng suggested. That property seems to be no longer needed for other browsers and breaking Safari. 

@vcanales or @jffng could you please confirm if the issue persists?

**Comment by vcanales** [comment]
Still seeing the problem, probably because the class applying the `min-width` is `.wp-column-block`.

The following fixes the issue for me:

```css
.club-post-list-pattern .wp-block-column {
	min-width: unset;
}
```

File: club/style.css [hunk 1]
```diff
@@ -105,14 +105,56 @@ GNU General Public License for more details.
...
+	white-space: nowrap;
+}
+
+.club-post-list-pattern .wp-block-read-more {
+	width: auto;
+	text-align: center;
+	word-break: keep-all;
+	width: 100%;
+	white-space: nowrap;
+}
```
Comment (vcanales) [inline club/style.css]: Noticed this duplicate `width` definition:

```suggestion
.club-post-list-pattern .wp-block-read-more {
	text-align: center;
	word-break: keep-all;
	width: 100%;
	white-space: nowrap;
}
```

**Comment by matiasbenedetto** [comment]
@vcanales I committed your suggestion.

Thanks, everyone for testing and suggesting fixes.

I'm merging this because the only concern is the button on safari. I'm creating a follow-up issue to check if the issue persists and eventually work on that: https://github.com/Automattic/themes/issues/6306

**Review by vcanales — APPROVED** [review]
> I'm merging this because the only concern is the button on safari. I'm creating a follow-up issue to check if the issue persists and eventually work on that.

Sure thing, I can take it over if you don't have Safari!

**Pull request merged**